python_modules/dagster/dagster/_core/executor/step_delegating/step_delegating_executor.py

```diff
@@ -209,6 +209,7 @@ def execute(self, plan_context: PlanOrchestrationContext, execution_plan: Execut
                                     step_handler_context.get_step_context(step.key),
                                     f"Including step {step.key} in the new run since it is not"
                                     f" currently running: {health_check.unhealthy_reason}",
+                                    EngineEventData(),
```

Comment:
the changes to the docker step health function surfaced an unrelated bug in this experimental feature here, this required argument was missing